src/Components/ProvisioningWizard/steps/FinishProgress/index.js

```diff
@@ -15,10 +15,11 @@ import { useWizardContext } from '../../../Common/WizardContext';
 import { useMutation, useQuery } from 'react-query';
 import { createAWSReservation, createNewPublicKey, fetchAWSReservation } from '../../../../API';
 import './styles.scss';
+import useInterval from '../../../Common/Hooks/useInterval';
 
 const pf_success_color_100 = '#3E8635';
 const pf_danger_color_100 = '#C9190B';
-const RESERVATION_POLLING_INTERVAL = 500;
+const POLLING_INTERVAL = [500, 1000, 1500, 2500, 4000, 6500, 10500, 17000];
```

Comment:
I am curious where did you get these numbers? Assuming these are milliseconds, 17 seconds feels like too much for what it's doing (reservation is just query in the DB which us sub 100ms request). I suggest to keep the backoff interval in 500ms - 5s or something like that.

Reply:
> I am curious where did you get these numbers? 

The numbers here are just sort of Fibonacci for the demonstration (yet, it's still POC :))

The main idea here is to make the backoff strategy dynamic with just a constant array, I think adding 100ms every interval is a bit naive and might add redundant roundtrips with no benefits to the user experience. we can use a mixed backoff strategy, which starts with a linear increase but ends with exponential backoff, i.e

500
600
700
800
900
1000
1250
1500
1750
2000
4000
8000
16000
32000
64000

Reply:
I'd agree with @amirfefer here, the initial is good to be linear, but then we can save some roundtrips (even tho those are fast, probably not necessary). I'd probably cut early (before the minute of wait (or add more frequent at the end) as the last two are quite large, but I guess that we should just try somthing off and if that does not work, then tweak it.

Reply:
One thought I had here was having some kind of identifier so we can track these requests belonging together. Maybe some additional header that we can track in our logger? Or do we think the `reservation_id` is enough?